ScoutSuite/providers/aws/aws_facade.py

```diff
@@ -0,0 +1,34 @@
+import boto3
```

Comment:
why not just `facade.py`?

Reply:
Yea I guess this would be more consistent with the codebase. It's so annoying though when you use the fuzzy search to open a file and it suggests 4 different files all named `provider.py` 😆